src/openms/include/OpenMS/METADATA/PeptideHit.h

```diff
@@ -19,10 +19,26 @@
 
 namespace OpenMS
 {
-  /**
-    @brief Representation of a peptide hit
+  class PeptideHit;
```

Comment:
Do you want to add a deprecated warning in doxygen to the old names then?
As long as doxygen treats them as separate entities it should be clearer then.